src/test/java/com/virtuslab/using_directives/custom/CommentsTests.java

```diff
@@ -19,6 +19,8 @@
   String specialComment = "//> using specialComment \"b\"";
   String specialComment2 = "//> using specialComment2 \"b\"";
   String keywordDirective = "using keywordDirective \"b\"";
+  String numericScalaVersionDirective = "//> using scalaFullNumeric 2.4, 7, \"true\"  \"wrong\" ";
```

Comment:
I do not see a test checking if `using scala 2.13.3` will fail.

Reply:
It fails. But I need to add a few better test cases…